### ISSUE TITLE
Weird name for init

### ISSUE THREAD

**Issue by alok** [David-OConnor/pyflow#23]
`pyflow init` seems to pick up the wrong name field from git config. My gitconfig:

```
[user]
	name = a
[merge "jupyternotebook"]
	driver = git-nbmergedriver merge %O %A %B %L %P
	name = jupyter notebook merge driver
```

It uses the `name` field under the `merge` table, which is clearly wrong.

**Comment by Boxfort** [comment]
I have proposed a fix for this in PR #24 :)

**Comment by David-OConnor** [comment]
@Boxfort diagnosed and fixed this in that PR, which is now merged. Will be in next release.

**Comment by David-OConnor** [comment]
Fixed in release 0.1.8.
